dashboard/client/src/service/log.ts

```diff
@@ -24,6 +24,7 @@ export const getLogDetail = async (url: string) => {
         const a = li.children[0] as HTMLAnchorElement;
         let href = a.href;
         if (
+          url !== "log_index" &&
```

Comment:
When access the logs index page, the front end will truncate the url link if it does not start with 'http'. This case helps to avoid url truncation like this:
![image](https://user-images.githubusercontent.com/16671583/205418198-ed2bdf3c-8b54-44e9-a605-e83de14c0038.png)

Reply:
oh I meant to update the code comment. sorry for confusion..